CalculatorInput.podspec

```
@@ -0,0 +1,17 @@
+Pod::Spec.new do |s|
+  s.name         = 'CalculatorInput'
+  s.version      = '2.0.1'
+  s.summary      = 'Venmo-style calculator keyboard for iOS'
+  s.description  = 'A pretty calculator keyboard for iOS. Use it as a standalone keyboard or use our CalculatorInputTextField.'
+
+  s.homepage     = 'https://github.com/venmo/VENCalculatorInputView'
+  s.license      = { :type => 'MIT', :file => 'LICENSE' }
+  s.author       = { 'Ayaka Nonaka' => 'ayaka@venmo.com' }
+  s.platform     = :ios, '6.0'
+  s.ios.deployment_target = '6.0'
```

The Deployment target in the project is 9.0

```suggestion
  s.ios.deployment_target = '9.0'
```